apps/pie-storybook/stories/pie-button.stories.tsx

```diff
@@ -170,24 +170,24 @@ const Template: TemplateFunction<ButtonProps> = ({
     formnovalidate,
     formtarget,
 }) => html`
-        <pie-button
-            size="${size}"
-            variant="${variant}"
-            type="${type}"
-            iconPlacement="${iconPlacement || nothing}"
-            ?disabled="${disabled}"
-            ?isLoading="${isLoading}"
-            ?isFullWidth="${isFullWidth}"
-            name=${name || nothing}
-            value=${value || nothing}
-            formaction=${formaction || nothing}
-            formenctype=${formenctype || nothing}
-            formmethod=${formmethod || nothing}
-            formtarget=${formtarget || nothing}
-            ?formnovalidate="${formnovalidate}">
-            ${iconPlacement ? html`<icon-plus-circle slot="icon"></icon-plus-circle>` : nothing}
-            ${slot}
-        </pie-button>`;
+<pie-button
+    size="${size}"
+    variant="${variant}"
+    type="${type}"
+    iconPlacement="${iconPlacement || nothing}"
+    ?disabled="${disabled}"
+    ?isLoading="${isLoading}"
+    ?isFullWidth="${isFullWidth}"
+    name=${name || nothing}
+    value=${value || nothing}
+    formaction=${formaction || nothing}
+    formenctype=${formenctype || nothing}
+    formmethod=${formmethod || nothing}
+    formtarget=${formtarget || nothing}
+    ?formnovalidate="${formnovalidate}">
+    ${iconPlacement ? html`<icon-plus-circle slot="icon"></icon-plus-circle>` : nothing}
+    ${slot}
```

Comment:
Just to add – I think it's to do with this section. If there's no icon, you'll get an extra line of whitespace in the docs. Can probably fix this by doing:
```suggestion
    ?formnovalidate="${formnovalidate}">${
    iconPlacement ? html`<icon-plus-circle slot="icon"></icon-plus-circle>` : nothing}
    ${slot}
```

Reply:
all fixed 😄 thank you! https://pr982-storybook.pie.design/?path=/docs/button--variants